pkg/experiment/metadata.go

```diff
@@ -124,15 +141,40 @@ func sslOptions(config MetadataConfig) *gocql.SslOptions {
 }
 
 // connect creates a session to the Cassandra cluster. This function should only be called once.
-func (m *Metadata) connect() error {
+func (m *Metadata) connect() *gocql.ClusterConfig {
 	cluster := gocql.NewCluster(m.config.CassandraAddress)
 
 	// TODO(niklas): make consistency configurable.
 	cluster.Consistency = gocql.LocalOne
 	cluster.SerialConsistency = gocql.LocalSerial
 
 	cluster.ProtoVersion = 4
-	cluster.Timeout = m.config.CassandraConnectionTimeout
+	cluster.ConnectTimeout = m.config.CassandraConnectionTimeout
+	cluster.Timeout = m.config.CassandraTimeout
+	cluster.IgnorePeerAddr = m.config.CassandraIgnorePeerAddr
+	cluster.DisableInitialHostLookup = !m.config.CassandraInitialHostLookup
+
+	return cluster
+}
+
+func (m *Metadata) createKeyspace() error {
+	config := m.connect()
```

Comment:
@squall0gd - why variable that represents datatabase connection is called `config`?